SPECS/btrfs-progs/btrfs-progs.spec

```diff
@@ -41,13 +43,14 @@ btrfs filesystem-specific programs.
 
 %build
 ./autogen.sh
-%configure \
-	--disable-zstd
-make DISABLE_DOCUMENTATION=1 %{?_smp_mflags}
+%configure CFLAGS="%{optflags} -fno-strict-aliasing" --with-crypto=libgcrypt --disable-python
```

Comment:
We dropped the disable-zstd.

if that option means “use system zstd” (not the embedded one) AND the option still exists, then can we put it back to be explicit that we are not using the built in version…. Helps with CVE disputes, and guarantees we’re not accidentally using some busted internal version